src/components/common/section-templates/shared/title-description-section/ViewTitleDescriptionSection.module.scss

```
@@ -35,11 +35,13 @@
 
 .title {
     @include type.h3-bold-uppercase;
-    word-wrap: break-word;
+    overflow-wrap: anywhere;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)